processing/src/test/java/io/druid/segment/CloserRuleTest.java

```diff
@@ -68,10 +65,10 @@ public void testAutoCloses() throws Throwable
     final CloserRule closer = new CloserRule(false);
     final AtomicBoolean closed = new AtomicBoolean(false);
     closer.closeLater(
-        new AutoCloseable()
+        new Closeable()
```

Comment:
probably none. I tried not to tweak the tests too much. I'll delete it or modify to make more obvious.